packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8531,6 +8531,33 @@ describe('plugin-meetings', () => {
           });
         });
       });
+
+      describe('#brb', () => {
+        it('send brb request', () => {
+          meeting.meetingRequest.locusDeltaRequest = sinon.stub().resolves();
+          meeting.locusUrl = 'locus url'
+          meeting.deviceUrl = 'device url'
+          meeting.selfId = 'self id'
+
+          const assertBrb = (enabled) => {
```

Comment:
@SomeBody16  I'm not sure if we should test the exact implementation details for this test case. We simply need to verify its functionality since we don’t know which internal requests API is used to make brb work.

For example, you can look at how the `toggleReactions` public API is tested. It doesn’t focus on the HTTP request details, and I haven’t found any similar approaches, and I believe this should also have something similar. Let’s ensure our tests are consistent with the existing logic for testing the public API.